Formula/open-coarrays.rb

```diff
@@ -0,0 +1,57 @@
+class OpenCoarrays < Formula
+  desc "open-source coarray Fortran ABI, API, and compiler wrapper"
+  homepage "http://opencoarrays.org"
+  url "https://github.com/sourceryinstitute/opencoarrays/releases/download/1.8.2/OpenCoarrays-1.8.2.tar.gz"
+  sha256 "dc5b98a62de85e38500f431b05290415d4a4a87001c0ba03fbe57403ad62010b"
+
+  head "https://github.com/sourceryinstitute/opencoarrays.git"
+
+  option "without-test", "Skip build time tests (not recommended)"
```

Comment:
(btw this is [the official GFortran back end](https://gcc.gnu.org/wiki/CoarrayLib) implementation of Coarray Fortran, added in the 2008 standard.)